janusgraph-core/src/main/java/org/janusgraph/core/EdgeLabel.java

```diff
@@ -55,4 +57,11 @@
      */
     Multiplicity multiplicity();
 
+    /**
+     * Collects all property constraints.
+     *
+     * @return a list of {@link PropertyKey} which represents all property constraints for a {@link EdgeLabel}.
+     */
+    Collection<PropertyKey> mappedProperties();
+
```

Comment:
`mgmt.addConnection()` returns an `EdgeLabel`, so wouldn't `mappedConnections()` be found on `EdgeLabel`?